### PR TITLE
README Updates

### PR DESCRIPTION
**Pull request by danielmorrison**
Simplify and try to explain what this is all about a bit better.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,37 +32,13 @@ the `--ruby_out` option). It does not generate Twirp services and clients; that
 
 ### Install the `protoc-gen-twirp_ruby` plugin
  
-Install the gem by adding it to your Gemfile:
+Run `gem install protoc-gen-twirp_ruby` or add it to your Gemfile:
```
Comment (darronschall) [inline README.md:35]: Probably worth keeping the note that this is intended as a development dependency, not a runtime one.

File: README.md [hunk 2]
```diff
@@ -25,37 +32,13 @@ the `--ruby_out` option). It does not generate Twirp services and clients; that
...
-group :development, :test do
-  "protoc-gen-twirp_ruby"
-end
-````
-
-Alternatively, install the gem on your system:
-
-```bash
-gem install protoc-gen-twirp_ruby
+gem "protoc-gen-twirp_ruby"
```
Comment (danielmorrison) [inline README.md:38]:
```suggestion
gem "protoc-gen-twirp_ruby", group: :development
```

File: README.md [hunk 3]
```diff
@@ -78,6 +61,26 @@ specifying `--twirp_ruby_opt=<option>` on the `protoc` command line.
...
+## Migrating from the Go module
+
+If you previously installed the `protoc-gen-twirp_ruby` Go module via the [Twirp-Ruby's Code Generation wiki page](https://github.com/arthurnn/twirp-ruby/wiki/Code-Generation)
+instructions, then you'll want to uninstall it before invoking the `protoc` command.
+
+```bash
+rm `go env GOPATH`/bin/protoc-gen-twirp_ruby
+```
+
+### Differences form the Go module
```
Comment (darronschall) [inline README.md:73]:
```suggestion
### Differences from the Go module
```

File: README.md [hunk 4]
```diff
@@ -25,49 +32,25 @@ the `--ruby_out` option). It does not generate Twirp services and clients; that
...
+Pass `--twirp_ruby_out` to `protoc` to generate Twirp-Ruby code:
 
 ```bash
 protoc --proto_path=. --ruby_out=. --twirp_ruby_out=. ./path/to/service.proto
 ```
 
 ### Options
 
-The plugin supports the following options to configure code generation. Pass options by
+You can configure the `protoc` code generation . Pass options by
```
Comment (darronschall) [inline README.md:53]:
```suggestion
You can configure the `protoc` code generation. Pass options by
```

File: protoc-gen-twirp_ruby.gemspec [hunk 5]
```diff
@@ -8,8 +8,8 @@ Gem::Specification.new do |spec|
   spec.authors = ["Darron Schall", "Daniel Morrison", "Chris Gaffney"]
   spec.email = "info@collectiveidea.com"
 
-  spec.summary = "A Ruby protoc plugin generator for Twirp-Ruby."
-  spec.description = "A Ruby protoc plugin generator for Twirp-Ruby."
+  spec.summary = "A protoc plugin, for generating Twirp-Ruby clients and/or services"
```
Comment (darronschall) [inline protoc-gen-twirp_ruby.gemspec:11]:
```suggestion
  spec.summary = "A protoc plugin for generating Twirp-Ruby clients and/or services"
```

**Pull request merged**